src/hooks/useUnsafeAccounts.ts

```diff
@@ -0,0 +1,39 @@
+// @ts-nocheck
+import { useEffect, useState } from "react";
+import { FLAGGED_ACCOUNT_STORAGE_ID } from "constants/settings";
+
+import { getFlaggedAccounts } from "helpers/getFlaggedAccounts";
+
+interface FlaggedAccount {
+  address: string;
+  tags: [string];
+}
+
+interface FlaggedAccounts extends Array<FlaggedAccount> {}
+
+export const useFlaggedAccounts = () => {
+  const [flaggedAccounts, setFlaggedAccounts] = useState<FlaggedAccounts>([
+    { address: "", tags: [] },
+  ]);
+
+  useEffect(() => {
+    const fetchFlaggedAccounts = async () => {
+      let accounts;
+
+      try {
+        accounts = await getFlaggedAccounts();
+      } catch (e) {
+        accounts =
+          JSON.parse(localStorage.getItem(FLAGGED_ACCOUNT_STORAGE_ID)) || [];
```

Comment:
per the Issue spec, if the request to stellar.expert doesn't resolve in time, let's try to grab a locally stored version of the list